### PR TITLE
Favorite Page bug fixes and visual / usability optimizations

### PR DESCRIPTION
**Pull request by AconexOfficial**
Favorite Page bug fixes and visual / usability optimizations

### Description

This is the Pull Request Template for the Oeffis Project

## Checklist

- [x] My code follows the style guidelines
- [ ] I have documented my code
- [x] My changes generate no new warnings
- [x] I have linked the Issue
- [x] The Pull Request covers all the Checkpoints from the tagged Issue
- [x] I have noted the hours of work i have put in the issue

## I have edited following Parts of the project

- [x] Frontend
- [ ] Backend
- [ ] Github
- [ ] Root

## If tests are required did you make them?

- [x] No tests required
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by LaraBiss** [comment]
If you use the arrows to swipe you can never get to the favorite routes (middle page). Think it could be better if the arrow leads you to the next page and not to the last.

**Comment by LaraBiss** [comment]
Currently the whole page is scrollable, wouldn't it be more efficient if just the items are scrollable and the navigation (radio buttons) are visible all the time? Currently the user must scroll all to the top to change to routes/trips if he has many favorite stations.

File: frontend/src/components/FavoriteLocationsComponent.tsx [hunk 1]
```diff
@@ -44,7 +44,9 @@ export const FavoriteLocationsComponent: React.FC<FavoriteLocationsComponentProp
                   onLocationSelected={props.onLocationSelected}
                   location={location} />
               ))
-              : <IonLabel>Keine favorisierten Locationn vorhanden</IonLabel>
+              : <IonLabel>
+                <p className={styles.no_favorites_text}>Keine favorisierten Locations vorhanden</p>
```
Comment (LaraBiss) [inline frontend/src/components/FavoriteLocationsComponent.tsx:48]: Using English and German. The headline of every Favorite Page is also in English. Use German instead.

**Review by LaraBiss — CHANGES_REQUESTED** [review]

**Comment by AconexOfficial** [comment]
> If you use the arrows to swipe you can never get to the favorite routes (middle page). Think it could be better if the arrow leads you to the next page and not to the last.

The arrows currently do not swipe at all, they are merely an indication, that it is possible to swipe and that there are multible pages. Can additionally add button functionality to the arrows aswell for going to next swiper page, though.

**Comment by AconexOfficial** [comment]
> Currently the whole page is scrollable, wouldn't it be more efficient if just the items are scrollable and the navigation (radio buttons) are visible all the time? Currently the user must scroll all to the top to change to routes/trips if he has many favorite stations.

I just found it jarring to have to click and drag the small item to scroll. Can also do this proposition instead. Would probable be a cleaner solution.

**Comment by LaraBiss** [comment]
> > If you use the arrows to swipe you can never get to the favorite routes (middle page). Think it could be better if the arrow leads you to the next page and not to the last.
> 
> The arrows currently do not swipe at all, they are merely an indication, that it is possible to swipe and that there are multible pages. Can additionally add button functionality to the arrows aswell for going to next swiper page, though.

In my application the arrows already have a button function. Clicking them leads you to the most left or the most right page.

**Comment by AconexOfficial** [comment]
> > > If you use the arrows to swipe you can never get to the favorite routes (middle page). Think it could be better if the arrow leads you to the next page and not to the last.
> > 
> > 
> > The arrows currently do not swipe at all, they are merely an indication, that it is possible to swipe and that there are multible pages. Can additionally add button functionality to the arrows aswell for going to next swiper page, though.
> 
> In my application the arrows already have a button function. Clicking them leads you to the most left or the most right page.

Hmm, interesting, since those are simply icons. Eitherway, I'm currently working on implementing the correct functionality.

File: frontend/src/components/FavoriteLocationsComponent.tsx (hunk 1 above)
Comment (AconexOfficial) [reply frontend/src/components/FavoriteLocationsComponent.tsx]: Fixed language aswell as implemented the other proposals

**Pull request merged**